build/yamls/base/conf/antrea-agent.conf

```diff
@@ -32,7 +32,10 @@ featureGates:
 #  Egress: true
 
 # Enable AntreaIPAM, which can allocate IP addresses from IPPools. AntreaIPAM is required by the
-# bridging mode and allocates IPs to Pods in bridging mode.
+# bridging mode and allocates IPs to Pods in bridging mode. It is also required to use Antrea for
```

Comment:
Do you think that a new line for each case (bridgingMode and secondaryNetwork) is more clear?

Reply:
I feel one paragraph is ok. I do not like short paragraphs.